Odata-docs/client/bulk-update-operations.md

```diff
@@ -0,0 +1,272 @@
+---
+title: "Bulk Update Operations in OData Client"
+description: "How to perform bulk update operations in OData client"
+
+author: ElizabethOkerio
+ms.author: elokerio
+ms.date: 6/29/2023
+ms.topic: article
+ 
+---
+# Bulk Update Operations
+**Applies To**: [!INCLUDE[appliesto-odataclient](../includes/appliesto-odataclient-v7.md)]
+
+OData Client [v7.17.0](https://www.nuget.org/packages/Microsoft.OData.Client/) introduced support for bulk update operations.
+
+## What is a bulk update operation 
+
+A bulk update operation is an operation that allows you to perform several operations (POST, PATCH and DELETE) on resources of the same type and all their related resources using a single request. This can be highly efficient in situations where you need to update multiple resources and wish to avoid sending separate requests to process each resource. [Learn more](https://devblogs.microsoft.com/odata/bulk-operations-support-in-odata-web-api/).
+
+## Sample bulk update requests
+
+### Model
+To create the various sample requests in this tutorial, the following model classes will be used: 
+
+``` csharp
+public class Customer
+{
+    public int Id { get; set; }
+    public string Name { get; set; }
+    public int Age { get; set; }
+    public List<Order> Orders { get; set; }
+}
+
+public class Order
+{
+    public int Id { get; set; }
+    public decimal Amount { get; set; }
+}
+```
+### Sample Request 1
+This sample request below demonstrates how to create a bulk update request in OData client to create 2 new customers in a single request:
+
+How to make the above request from OData client.
+
+```csharp
+var context = new Container(new Uri("http://localhost:6285/odata/"));
+
+var customer1 = new Customer
+{
+    Id = 1,
+    Name = "Customer1",
+    Age = 26
+};
+
+context.AddToCustomers(customer1);
+
+var customer2 = new Customer
+{
+    Id = 2,
+    Name = "Customer2",
+    Age = 34
+};
+
+context.AddToCustomers(customer2)
+
+DataServiceResponse response = await context.BulkUpdateAsync(customer1, customer2);
+```
+
```

Comment:
Between the C# code block and the request payload that is generated behind the scenes, you should add something like:

The code above generates the following bulk update request payload: